federationclient.go

```diff
@@ -42,16 +46,21 @@ func (ac *FederationClient) doRequest(ctx context.Context, r FederationRequest,
 		return err
 	}
 
+	logger.Infof("Outgoing request %s %s", req.Method, req.URL)
```

Comment:
Why does one log line say "Outgoing request" and the other say "Sending request"?

Reply:
Because I changed one to "Outgoing" and forgot to change the other.